### ISSUE TITLE
Why CircleCI badge report failure when it is not so?

### ISSUE THREAD

**Issue by pllim** [spacetelescope/notebooks#91]
@obviousrebel , do you know why CircleCI badge on the repo README is red even when https://circleci.com/gh/spacetelescope/notebooks/43 (the latest merge commit at the time of writing) shows green?

**Comment by obviousrebel** [comment]
It failed the [build and check job](https://circleci.com/gh/spacetelescope/notebooks/41) but passed the [deploy job](https://circleci.com/gh/spacetelescope/notebooks/43). Both of these jobs are in one workflow so if one fails, the overall job is failing.

**Comment by pllim** [comment]
Hmm is there a way to force it to re-run the whole thing for a green badge? Or more importantly, did it deploy the right stuff, given the "build and check" one failed?

**Comment by obviousrebel** [comment]
It shouldn't start the deploy stage when the build fails, I'll have to do some investigating. If it did, it doesn't remove anything. Just adds/overwrites changed files so the gh-pages branch should be fine as it would still have the html files from the previous commit.

**Comment by obviousrebel** [comment]
Looks like the deploy job 43 was actually for build_and_check job 39. The workflows must have been triggered fairly close together. So the workflow containing the jobs 39 and 43 passed, but the latest workflow containing job 41 failed giving the failed status badge.

tldr:
Everything is working as it should. Deploy job only runs if build_and_check is successful, badge shows latest workflow status for master branch.

**Comment by pllim** [comment]
How do we make it green again? Is something broken?

**Comment by pllim** [comment]
I mean, not the deploy process, but is one of the notebooks broken?

**Comment by obviousrebel** [comment]
It is green currently as the last two workflows have passed. The [last failure](https://circleci.com/gh/spacetelescope/notebooks/41) was because a cell timed out in the align_mosaics notebook on a network call.
